lib/generators/boring/brakeman/install/install_generator.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+module Boring
+  module Brakeman
+    class InstallGenerator < Rails::Generators::Base
+      desc 'Adds Brakeman to the application'
+      source_root File.expand_path('templates', __dir__)
+
+      RUBY_VERSION_FILE = '.ruby-version'
+
+      DEFAULT_RUBY_VERSION = '.ruby-version'
+
+      class_option :skip_config, type: :boolean, aliases: '-c',
+                                                desc: 'Skip adding Brakeman configuration. Defaults to false'
+
+      class_option :github, type: :boolean, aliases: '-gh',
+                                                desc: 'Add GitHub Actions configuration. Defaults to false'
+
+      class_option :gitlab, type: :boolean, aliases: '-gl',
+                                                desc: 'Add GitLab CI configuration. Defaults to false'
+
+      class_option :ruby_version, type: :string, aliases: '-v',
```

Comment:
Let's remove DEFAULT_RUBY_VERSION because we don't want to use static value here.

Also update description to "Tell us the ruby version which you use for the application. Defaults to the ruby version in your app's Gemfile"